pkg/math/voteweighted/threshold.go

```diff
@@ -0,0 +1,87 @@
+package voteweighted
+
+import (
+	"math/big"
+
+	"cosmossdk.io/math"
+)
+
+var (
+	Zero       = big.NewInt(0)
+	Two        = big.NewInt(2)
+	OneMillion = big.NewInt(1_000_000)
+)
+
+// SignsDiffer determines whether newPrice and newPrime move in different directions relative to old.
```

Comment:
nit: you might have to explain whether this function returns true or false if one or both of the values is zero. It seems to do this weird thing where if exactly one is zero, it returns true, but false if both are zero